pkg/config/kind.go

```diff
@@ -35,24 +35,24 @@ const (
 
 // Name of all supported aspect kinds.
 const (
-	AccessLogsKindName          = "access-logs"
-	ApplicationLogsKindName     = "application-logs"
-	AttributeGenerationKindName = "attribute-generation"
-	DenialsKindName             = "denials"
-	ListsKindName               = "lists"
-	MetricsKindName             = "metrics"
-	QuotasKindName              = "quotas"
+	AccessLogsKindName      = "access-logs"
+	ApplicationLogsKindName = "application-logs"
+	AttributesKindName      = "attributes"
```

Comment:
they could be other preprocess aspect kinds. this is just _a_ kind of processor that generates attributes. when we identify other proprocess needs, they would be a separate preprocess aspect, with a different kind name.